### ISSUE TITLE
Update banner on Listings page

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#821]
In an effort to make the [Listings](https://sites-public-tj3gg4i5eq-uc.a.run.app/listings) page more visually appealing and accessible, we would like to modify the banner in the following ways:

1) Change background from gray to green.
2) Change text from black to white.
3) Add 'Find rentals for me' button.
![listing_banner](https://user-images.githubusercontent.com/394908/145123047-d56c85b5-5226-4c22-8f87-4ee9ec8a897b.png)

Design is available in [Figma](https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497) (02.1).
